test/README.md

```diff
@@ -215,6 +215,13 @@ Change directory to the test directory:
 cd test
 ```
 
+To setup up local kubernetes cluster for drenv, run once before starting any environment:
```

Comment:
This does not set up local kubernetest cluster. It sets up the host for running the specified environment.

Reply:
Updated as suggested.